protobuf-definitions/README.md

```diff
@@ -26,3 +26,14 @@ On Windows & Linux: [See here](https://github.com/google/protobuf/blob/master/sr
 1. Install pre-requisites.
 2. Un-comment line 4 in `make.bat`, and set to correct Grpc.Tools sub-directory.
 3. Run `make.bat`
+4. In the generated `UnityToExternalGrpc.cs` file in the `CommunicatorObjects` folder, you will need to add on top of the file the line
```

Comment:
Say where `CommunicatorObjects` folder is (i.e. in UnitySDK). 

Also change  "to add on top of the file the line"
to
"to add the following to the beginning of the file"